packages/datetime/src/common/months.ts

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright 2017 Palantir Technologies, Inc. All rights reserved.
+ * Licensed under the BSD-3 License as modified (the “License”); you may obtain a copy
+ * of the license at https://github.com/palantir/blueprint/blob/master/LICENSE
+ * and https://github.com/palantir/blueprint/blob/master/PATENTS
+ */
+
+// tslint:disable object-literal-sort-keys
+export const Months = {
```

Comment:
sweet, even better. ill leave it as a normal enum until we need to do the `1-index` variant then, cool?